test/basic.ml

```diff
@@ -142,43 +230,97 @@ let%expect_test "utf8" =
   [%expect
```

Comment:
It'd be nice to update the utf8 test so that the input contains uchar spanning multiple bytes.

Reply:
Done!